.appveyor.yml

```diff
@@ -11,7 +11,7 @@ environment:
     - build_type: windows32_cocos_new_test
     - build_type: windows32
     - build_type: android_cpp_tests
-    - build_type: android_cocos_new_test
+#    - build_type: android_cocos_new_test
```

Comment:
what's the reason to remove `android_cocos_new_test` task?

Reply:
To speedup appveyor builds.
android_cocos_new_test is run in travis as well.
I see that appveyor is much slower than travis. 
p.s.
All appveyor build configurations except windows32 were added by me 6 months ago :)

Reply:
that's fine :)